### PR TITLE
[SOL-2138] Multi-use coupon max_uses field displays incorrect value fix

### PR DESCRIPTION
**Pull request by iivic**
@vkaracic @marjev @mjfrey 
https://openedx.atlassian.net/browse/SOL-2138

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @iivic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?number=1020&repo=edx%2Fecommerce"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

File: ecommerce/static/js/test/specs/views/coupon_form_view_spec.js [hunk 1]
```diff
@@ -193,6 +193,12 @@ define([
                     view.$('[name=voucher_type]').val('Multi-use').trigger('change');
                     expect(view.$('[name=max_uses]').val()).toBe('');
                     expect(view.$('[name=max_uses]').attr('min')).toBe('2');
+                    view.model.set('max_uses', 5);
+                    view.editing = true;
+                    view.render();
+                    view.$('[name=voucher_type]').val('Multi-use').trigger('change');
+                    expect(view.$('[name=max_uses]').val()).toBe('5');
+                    expect(view.$('[name=max_uses]').attr('min')).toBe('2');
```
Comment (vkaracic) [inline ecommerce/static/js/test/specs/views/coupon_form_view_spec.js]: Please move this to a separate test unit.

**Review by vkaracic — CHANGES_REQUESTED** [review]

**Review by marjev — APPROVED** [review]

**Comment by marjev** [comment]
LGTM, just address the comment Vedran left you.

**Comment by iivic** [comment]
I created new coupon data `enrollmentMultiUseCouponData` and formatted the whole `coupons.js` file.

File: ecommerce/static/js/test/mock_data/coupons.js [hunk 2]
```diff
@@ -317,6 +338,7 @@ define([], function(){
         'enrollmentCodeCouponData': enrollmentCodeCouponData,
         'enrollmentCodeCouponModelData': enrollmentCodeCouponModelData,
         'enrollmentCodeVoucher': enrollmentCodeVoucher,
+        'enrollmentMultiUseCouponData': enrollmentMultiUseCouponData,
         'lastEditData': lastEditData,
         'percentageDiscountCodeVoucher': percentageDiscountCodeVoucher,
         'valueDiscountCodeVoucher': valueDiscountCodeVoucher,
```
Comment (vkaracic) [inline ecommerce/static/js/test/mock_data/coupons.js:344]: Why did you move all this around?

**Review by vkaracic — APPROVED** [review]
LGTM although all those changes weren't necessary (the moving parts around). Also the file containing all the coupon mocks is heavily bloated with duplicates, but that's outside of the scope of this work, nevertheless we should DRY it sometimes in the future.

File: ecommerce/static/js/test/mock_data/coupons.js (hunk 2 above)
Comment (iivic) [reply ecommerce/static/js/test/mock_data/coupons.js:344]: I didn't move the code just aligned everything as it should be...
